banzai_nres/utils/wavelength_utils.py

```diff
@@ -28,6 +28,16 @@ def identify_features(data, err, mask=None, nsigma=2., fwhm=6.0, **kwargs):
     return features
 
 
+def index_of_refraction_Edlen(vacuum_wavelength):
+    """
+    The original 1966 Edlen Equation (Bengt Edlén 1966 Metrologia 2 71, https://doi.org/10.1088/0026-1394/2/2/002)
+    :param vacuum_wavelength: wavelength in vacuum
+    :return: index of refraction at that wavelength.
+    """
+    sig2 = (1E4/vacuum_wavelength)**2
+    return 1 + 1E-8 * (8342.13 + 2406030/(130 - sig2) + 15997/(38.9 - sig2))
```

Comment:
Pep8. Also why do you multiply by 1?

Reply:
where is the multiply by 1? I see the addition by 1, but not the multiply by 1.

Is the PEP8 in the function name? Edlen to lowercase edlen?

Reply:
doesn't that do return 1 * 1e-8?

Reply:
pep8 in the spacing

Reply:
oh ok, thanks for pep8, I see now.

I think I am misunderstanding: `1 + 1E-8 * (8342.13 + 2406030/(130 - sig2) + 15997/(38.9 - sig2))` will return something slightly larger than 1, not 1*1e-8